docs/pages/es/installation/windows.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: page
+title: Windows
+permalink: es/installation/platforms/windows
+---
+
+## **Inicio rápido con Windows**
+Para configurar y ejecutar Meshery en Windows:
+
+1. <a href="#step1">Configurar Windows y habilitar Docker </a>
+2. <a href="#step4">Instalar un cluster Kubernetes</a>
+3. <a href="#step5">Instalar Meshery</a>
+
+### **Compatibilidad**
+
+Las siguientes versiones mínimas de Compilación de Windows son requeridas:
+<table id="compatibility-table">
+  <tr>
+    <th id="model">Nombre</th>
+    <th id="model">Versión</th>
+  </tr>
+  <tr>
+    <td><a href="#wsl1">WSL1</a></td>
+    <td><b>x64</b> - Windows 7 </td>
+  </tr>
+  <tr>
+    <td><a href="#wsl2">WSL2</a></td>
+    <td><b>x64</b> - Version 1903, Compilación 18362; <b>ARM 64</b> - Version 2004, Compilación 19041</td>
+  </tr>
+  <tr>
+    <td><a href="https://docs.microsoft.com/en-us/windows/wsl/release-notes#build-18945">Kernel Personalizado</a></td>
+    <td>Compilación 18945</td>
+  </tr>
+  <tr>
+    <td><a href="https://docs.microsoft.com/en-us/windows/wsl/release-notes#build-19013">Kernel con módulos requeridos K8s</a></td>
+    <td>Compilación 19013</td>
+  </tr>
+</table>
+
+**Nota**
+<br />Ejecute el siguiente comando en Powershell para comprobar la compilación y la versión de Windows:
+```powershell
+[System.Environment]::OSVersion.Version
+```
+
+### **Pasos**
+Realice los siguientes pasos en orden:
+
+#### 1. <a name="step1" href="https://docs.microsoft.com/en-us/windows/wsl/install-win10"><b>Instalar el subsistema de Windows para Linux (WSL)</b></a>
+
+Abra la Powershell en modo administrador y ejecute:
+
+```powershell
+dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
+Reiniciar-Computadora
```

Comment:
i don't think this needs to be translated, in fact it doesn't work on my machine

Reply:
Perdón se refiere a reiniciar compitadora? Saludos

Reply:
Si, funciona ese cmdlet?

Reply:
Es verdad, lo siento, ya lo corrijo... Saludos